reana_cluster/backends/kubernetes/templates/deployments/db-template.yaml

```diff
@@ -18,12 +18,25 @@ spec:
         env:
         - name: TZ
           value: "Europe/Zurich"
-        - name: POSTGRES_USER
-          value: reana
         - name: POSTGRES_DB
           value: reana
+        {% if DEBUG %}
+        - name: POSTGRES_USER
+          value: reana
         - name: POSTGRESQL_PASSWORD
```

Comment:
Would it be possible to use the same prefix for all vars? either `POSTGRES` or `POSTGRESQL`?